index.src.html

```diff
@@ -869,9 +980,14 @@ <h3 id="information-leakage">Information Leakage</h3>
   <a>"<code>unsafe-url</code>"</a> might leak the origin and the URL of
   a secure site respectively via insecure transport.
 
-  Those two policies are include in the spec nevertheless to lower the friction
+  Those two policies are included in the spec nevertheless to lower the friction
   of sites adopting secure transport.
 
+  Authors can use the policy states <a>"<code>safe-origin</code>"</a>,
+  <a>"<code>safe-origin-when-cross-origin</code>"</a> and
+  <a>"<code>no-referrer</code>"</a> to prevent sensitive information from
```

Comment:
I took a stab at rephrasing this in https://github.com/w3c/webappsec-referrer-policy/pull/56/commits/2bce42c18e149734ab2bfb68993393cfcbc8f74f